### DISCUSSION TITLE
What is the "login" keychain password?

### DISCUSSION THREAD

**Discussion by Gwhain** [Chia-Network/chia-blockchain#1369] [Q&A]
Hi,

I am trying to create a plot but it hangs and when I look at the logs it says access denied.
I guess I put the wrong password.
What is the "login" keychain password? Is it the 10 digits private key?
I was not asked to create a password so I really wonder.

Many thanks,
GWhain

**Comment by Gwhain** [comment]
Ok, this seems to be the password to lock the computer.

**Reply by hoffmang9** [reply]
Chia keys are stored on the OS Keyrings and thus use your login password on Mac and Windows.

**Reply by Gwhain** [reply]
Got it thanks!
